src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -72,6 +73,14 @@ protected override void Initialize()
 
         protected Task LaunchScript(RequestContext<object> requestContext)
         {
+            // If this is a run without debugging session, disable all breakpoints.
+            if (this.noDebug)
+            {
+                Task disableBreakpointsTask = 
+                    this.editorSession.DebugService.DisableAllBreakpoints();
+                Task.WhenAll(disableBreakpointsTask);
```

Comment:
Yeah, it would be much easier to make that LaunchScript method async but I was worried about the ripple effects of that.  :-)